### PR TITLE
overlayrenderer: Avoid null access

### PR DESCRIPTION
**Pull request by popcornmix**

### CODE REVIEW

**Comment by popcornmix** [comment]
Fairly minor - I hit this after switching between different build methods.
I think perhaps the font hadn't been installed.
It was resolved with a clean build, but seems worth protecting against a null access.

@FernetMenta okay?

jenkins build this please

File: xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererGUI.cpp [hunk 1]
```diff
@@ -159,6 +159,9 @@ void COverlayText::PrepareRender(const std::string &font, int color, int height,
   if (!m_layout)
     m_layout = GetFontLayout(font, color, height, style, fontcache, fontbordercache);
 
+  if (m_layout == NULL)
+    return;
```
Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererGUI.cpp]: this should not happen, right? why not logging an error?

**Comment by popcornmix** [comment]
@FernetMenta Added log message for failure.

**Comment by FernetMenta** [comment]
thanks

**Pull request merged**